shared_model/packages/python/README.md

```diff
@@ -1,4 +1,5 @@
 # Python client library example
+***Please read `setup.py` and take a look at the branch that are you going to use.***
```

Comment:
I think it would be better to note that default branch is develop and if you somebody wants to use another branch offer to edit setup.py.